content/en/blog/2023/notation-completes-security-audit.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Announcing results of Notation security audit 2023
+author:  "Notation team"
```

Comment:
Should this be:
- Notary Project team?
- Notary Project?
- Notary team?

In the first sentence you say 
> In early 2023, Notary Project, under the guidance of [Cloud Native Computing Foundation](https://cncf.io/) began work with [Ada Logics](https://adalogics.com/)

Whichever you choose, you need to be consistent with the `author` and how you're referring to the team

Reply:
I would prefer `Notary Project maintainers` as this term has been mentioned multiple times in this article.